src/app/services/fake server/in-memory-data.service.ts

```diff
@@ -24,6 +29,167 @@ export class InMemoryDataService implements InMemoryDbService {
                     email:'john@gmail.com',
                     password:'testPass123!2' 
                 }
+            ],
+        
+            wellnessRatings:[              
+                { id: 1,
+                    userId: 1,
+                    date:'02-11-2023',
```

Comment:
it's more common to get an ISO date string back from an API. You may want to change to something like that for your test/mock data.